haiku-apps/minimizeall/minimizeall-1.0.0.recipe

```diff
@@ -1,15 +1,18 @@
 SUMMARY="Minimize all your apps with the press of a button!"
-DESCRIPTION="
+DESCRIPTION="
 MinimizeAll is a small app that minimizes all your apps with the press of a \
 button on the Deskbar, or with a keypress. The app contains two parts: A \
 standalone app that minimizes the whole app, and a replicant for your Deskbar. \
 Both do the same: Minimize all your apps!"
-HOMEPAGE="https://github.com/HaikuArchives/MinimizeAll/"
+HOMEPAGE="http://github.com/HaikuArchives/MinimizeAll"
```

Comment:
Should be
```
HOMEPAGE="https://github.com/HaikuArchives/MinimizeAll"
```
(you dropped the s in https.)
Good catch, BTW, for removing the trailing slash ;-)